Gemfile.lock

```diff
@@ -236,7 +236,7 @@ GEM
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
     rainbow (2.1.0)
-    rake (11.3.0)
+    rake (12.0.0)
```

Comment:
Probably something specific to my local setup but I started getting this error
```
Gem::LoadError: You have already activated rake 12.0.0, but your Gemfile requires rake 11.3.0. Prepending `bundle exec` to your command may solve this.
```

Updating rake to 12.0.0 (latest version) solved it, and I figured wouldn't hurt anything.